pallets/protocol-fee/src/lib.rs

```diff
@@ -208,13 +214,27 @@ impl<T: Config> Module<T> {
     /// Withdraws a precomputed fee from the current payer if it is defined or from the current
     /// identity otherwise.
     fn withdraw_fee(account: T::AccountId, fee: BalanceOf<T>) -> WithdrawFeeResult<T> {
+        // Check if the `account` is being subsidised.
+        let subsidiser = T::Subsidiser::check_subsidy(&account, fee, None)
```

Comment:
How come we're using `None` for the pallet here? Protocol fees are charged by extrinsics which are in certain pallets, so ostensibly we should be able to use the pallet name here too?

Reply:
The pallet restriction is already done in the transaction fees pallet's `SignedExtension`.  Originally I had wanted to get the current pallet name from the `permissions` pallet, but it's signed extension runs after the transaction fees signed extension, so `CurrentPalletName` was empty.

I talked with Adam at yesterday's stand-up.  We had decided that it wasn't needed to check the pallet name a second time for protocol fees.